pom.xml

```diff
@@ -131,6 +131,11 @@
       <version>1.19.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>2.6.0</version>
+    </dependency>
```

Comment:
Why does a pull request that states it is adding a limit to an API endpoint require a new Kafka dependency?

Reply:
I removed it